lib/backend/k8s/thirdparty/global_config.go

```diff
@@ -0,0 +1,78 @@
+package thirdparty
+
+import (
+	"encoding/json"
+
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/pkg/api/meta"
+	"k8s.io/client-go/pkg/api/unversioned"
+	"k8s.io/client-go/pkg/runtime/schema"
+)
+
+type GlobalConfigSpec struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
```

Comment:
All of our global config is a string - so that should be fine.   It's up to Felix to interpret as required.

The mapping to a specific data type is handled in the front end side of the config (or at least will be once we map through all of the possible values to the front end).